### ISSUE TITLE
세션이 전부 사라졌습니다

### ISSUE THREAD

**Issue by DevKiHyun** [AiHackathon2018/AI-Vision#123]
세션 현황 부분에 전부 다 사라졌습니다.....

**Comment by cyc1am3n** [comment]
저는 크레딧도 사라지고 재로그인도 안 되네요ㅠㅠ

**Comment by DaeHwanGi** [comment]
저는 세션이 모두 사라져서 로그아웃을한 이후로 로그인이 되지 않고 있습니다.

**Comment by DaehunGwak** [comment]
다 같은 현상을 겪고 있는 것 같아요

**Comment by sunnys-lab** [comment]
네 그러게요... 이번 대회는 대회의 원래 자체 목적보다는 저희를 NSML Closed Beta Tester로 부려먹는 느낌이 강합니다.. ㅠ.ㅠ

**Comment by DevKiHyun** [comment]
일주일째 딥러닝 작업은 못하고 nsml 작업만 계속 하는것 같네요...

**Comment by nsml-admin** [comment]
30분 정도의 web login 오류가 있습니다. 
오류시간: 1월 8일,  13:45  ~14:13

웹 login 유실로 세션이 보이지 않은 것이며, 학습중인 session은 정상 동작합니다. 

불편을 드려서 죄송합니다.

**Comment by sunnys-lab** [comment]
@nsml-admin 고생이 많으십니다. 빠른 처리 감사드립니다. ^^
